x/time/time.go

```diff
@@ -0,0 +1,17 @@
+package time
```

Comment:
This file will be merged in PR1, its just here until that one is merged

Reply:
think you need to rebase this and the other prs

Reply:
yep was just waiting for PR1 to get approved so I could do that, thanks for stamping it! Will rebase everything.